### DISCUSSION TITLE
Question about suspended cards

### DISCUSSION THREAD

**Discussion by Raytracer42** [mortii/anki-morphs#288] [Q&A] [answered]
Sorry, I'm new to the addon.

So, if I answer the card, the target morph will be moved to learning, right? And no new cards with this target morph will show up? (Assuming I have "skip already seen" enabled). Or am I missing something? What exactly happens to skipped cards?

Also, let's say, after a bit of studying I realized that the sentence on the card just doesen't stick in my memory and I suspend it. Will I encounter previously skipped cards with the same target morph again or do I have to manually search for one?

**Comment by mortii** [comment, marked as answer]
> So, if I answer the card, the target morph will be moved to learning, right? And no new cards with this target morph will show up? (Assuming I have "skip already seen" enabled). Or am I missing something?

That's right. If you answer one card that has a particular focus morph and you have `skip already seen` enabled, then the subsequent cards you encounter on the same day with that same focus morph will be buried (suspended for one day). If you don't want to see those buried cards the next day you have to enable one or more of these options, and make sure to recalc before your study session the next day:

![Screenshot from 2024-09-15 11-00-16](https://github.com/user-attachments/assets/64a56656-f379-461e-bd98-7e20e0f849cc)

> Also, let's say, after a bit of studying I realized that the sentence on the card just doesen't stick in my memory and I suspend it. Will I encounter previously skipped cards with the same target morph again or do I have to manually search for one?

Yes, if you enable the [ignore content in suspended cards](https://mortii.github.io/anki-morphs/user_guide/setup/settings/preprocess.html) option, then you can suspend a card, recalc, and all the unstudied cards that has that as their only unknown morph will now be moved to the start of the card queue. If you use the `suspend new cards with only known morphs` option then this kind of approach can get a little murky, and you will probably get some unintended consequences, so I recommend unchecking that.

With all that said, I personally use approaches that are deliberately anti-perfectionism and anti-worry since I tend to fall into those traps often. If I have a leech card where I don't really learn the focus morph then I just delete it, then I don't have to worry about any unintended consequences of suspending it. If I encounter a card that has words that I known in theory, but not in practice i.e., cards that have 1.5 unknowns, then I just throw the whole sentence into chatgpt and ask it to break down it down into it's compenents, and then I copy paste that onto the back of the card.

Did any of that make sense? Do you think some parts of the guide need more clarification?

**Reply by Raytracer42** [reply]
Yeah I think that answers my question. Also I'm sorry for mixing up burying and suspending. These terms mean different things in Anki.

I think my main cofusion came from the assumption that _by default_ AnkiMorphs will only show one card with a target morph and all the skipped (or technically "buried for a day") cards are gone "forever" and will not show up in front of the new cards queue.  

I guess that becomes clear after you've used the addon for at least a couple of days. But I'm only planning to use it after finishing another deck that doesen't require it and I'm currently just collecting and preparing sub2srs decks.

Regarding perfectionism. I guess I'm overthinking. In my mind you sorf of commit to learn this particular morph with this particular sentence. And if for some reason you later decide that you don't like this sentence and get rid of it (either by suspending or deleting the card before the morph is marked as "known") you also giving up on that morph. (But I guess it's not really true if you don't have any settings shown in your screeenshot enabled).

**Reply by mortii** [reply]
> I guess that becomes clear after you've used the addon for at least a couple of days.

Yeah, sorry for not having a clear enough guide. I think the best advice I can give is to create a backup of your collection and your sub2srs deck and then try a bunch of settings over a couple of days to see if you can make the add-on do what you want it to do, and if not, revert to your backup(s). 

If you have any other questions, or suggestions, let me know!
